test/rts_gmlc.jl

```diff
@@ -0,0 +1,31 @@
+"""
+Use PSCB to build RTS-GMLC System and add outage data
+as a Supplmental Attribute
+"""
+function get_rts_gmlc_outage()
```

Comment:
This isn't a separate test, it's a utility file imported by runtests.jl. Yes, and we won't need this modification once we have supplemental attributes in case builder.